php/libraries/User.class.inc

```diff
@@ -1,12 +1,13 @@
 <?php declare(strict_types=1);
 
 /**
- * This file contains the Loris User class
+ * This file contains the Loris user class
  *
- * PHP Version 7.4+
+ * PHP Version 7
  *
  * @category Main
  * @package  Main
+ * @author   Alex Zijdenbos <zijdenbos@example.com>
```

Comment:
changes like this (by your IDE) are cluttering -- we'd like to see just what you changed (as discussed Friday).   
@NadaElmasry if you need help with next steps here please let us know ASAP